toolchain/gcc/Config.in

```diff
@@ -17,6 +17,10 @@ choice
 
 	config GCC_USE_VERSION_8
 		bool "gcc 8.x"
+
+	config GCC_USE_VERSION_9
+		bool "gcc 9.x"
+		depends on !arc
```

Comment:
It seems arc is available since gcc 8.3 thus no need to exclude gcc 9

https://github.com/openwrt/openwrt/pull/1803#issuecomment-462334890
https://github.com/openwrt/openwrt/commit/343b36ffc79f9506586b5263b5d840dce0d801e0

Reply:
I pushed the change up. Thanks!!